### DISCUSSION TITLE
O que acham desse ebook sobre portfolios?

### DISCUSSION THREAD

**Discussion by gusfoca** [frontendbr/forum#2243] [Perguntas e Respostas]
Bom dia! Encontrei esse ebook do Josh Comeau sobre construção de portfolios para frontend:

https://www.joshwcomeau.com/effective-portfolio/

O que acham? Ele serve pro mercado frontend do Brasil? Há algo a ser adaptado?

Desde já agradeço os comentários!

**Comment by theandersonn** [comment]
Peguei uma copia aqui e li por alto.
Gostei do conteúdo e faz sentido sim.

Achei bem interessante as dicas sobre curadoria de projetos, caso o desenvolvedor seja iniciante e não tenha projetos reais, vale sim colocar projetos ficticios para a demonstração do a pessoa sabe.

Perguntas que normalmente são feita em entrevistas é:
- Entre os projetos que vc participou, qual vc considera mais importante e por que?
- Qual foi o mais desafiador
- Qual foi o aprendizado no projeto x

Então a montagem de um portfólio depende tbm do interesse, vender projetos para clientes, demonstrar conhecimento em entrevistas numa empresa e a partir de então pensamos em como apresentar e defender nosso conhecimento.
